solidity/test/isms/OPStackIsm.t.sol

```diff
@@ -204,60 +241,18 @@ contract OPStackIsmTest is Test {
             encodedHookData
         );
 
-        assertTrue(
-            opISM.verifiedMessageIds(
-                messageId,
-                address(this).addressToBytes32()
-            )
-        );
-
+        assertTrue(opISM.verifiedMessageIds(messageId) >> 255 == 1);
```

Comment:
use bit lib?

Reply:
and probably add unit tests for just bit lib

Reply:
added